pandas/meson.build

```diff
@@ -0,0 +1,51 @@
+incdir_numpy = run_command(py,
+  [
+    '-c',
+    'import os; os.chdir(".."); import numpy; print(numpy.get_include())'
+  ],
+  check: true
+).stdout().strip()
+
+inc_np = include_directories(incdir_numpy)
```

Comment:
The NumPy headers are not guaranteed to be in a subdirecrory of the Meson project, thus this works only in some special cases. I haven't tried, but wouldn't it be better to write something like:
```meson
np_dep = declare_dependency(include_directories: incdir_numpy)
```
and change the places where `inc_np`is used accordingly?